packages/cfd/src/Containers/__tests__/cfd-password-modal.spec.tsx

```diff
@@ -411,9 +490,15 @@ describe('<CFDPasswordModal/>', () => {
     });
 
     it('should display IcCfds icon in Success Dialog', async () => {
-        const store = mockStore(mockRootStore);
-
-        store.client.account_status = { status: ['mt5_password_not_set', 'dxtrade_password_not_set'] };
+        const store = mockStore(mock_root_store);
```

Comment:
can move the store initialisation to the outer scope unless custom store values needed

Reply:
The store value is modified in every test case so if we do it globally the other test cases will fail